app/models/project.rb

```diff
@@ -93,6 +107,10 @@ def calculator
     @calculator ||= ScoreCalculator.new(self)
   end
 
+  def open_issues
```

Comment:
ScoreCalculator wants the data in a slightly different usage, so the Issues are created by looking them up from the API then ScoreCalculator uses what we've got in the database to do its thing.